### ISSUE TITLE
accessibility issues

### ISSUE THREAD

**Issue by StenSeegel** [HAWK-Digital-Environments/HAWKI#155]
Our accessibility officer provided us with a demonstration of how to access HAWKI using a screen reader. We’ve identified several areas where we can improve accessibility:

- The site header lacks a language tag.
- The language tag in HAWKI V1 remains unchanged when the language is changed. In HAWKI 2, the language tag should be dynamically generated (preferably based on the client’s system language and changed when HAWKI language settings change).
- The login form lacks an H1 heading, making it difficult to identify the login field when using a screenreader and keyboard navigation.
- The HAWKI 2 interface immediately transitions to the chat input when using keyboard navigation (an onFocus property is set).
- The send button is not accessible through keyboard navigation (maybe because it’s actually a div).
- Keyboard navigation doesn’t allow switching between different chats.

To prevent similar issues in the future, he offered us a group workshop so we can better incorporate accessibility features into our future development. Let's talk about it when we meet!
